src/glang/codegen/type_conversions.py

```diff
@@ -197,7 +197,7 @@ def last_type() -> Type:
                 context,
                 src.format_for_output_to_user(),
                 dest_type.format_for_output_to_user(True),
-                maybe_missing_borrow,
+                maybe_missing_borrow=maybe_missing_borrow,
```

Comment:
I just don't like this. The type name adds nothing caller-side.

Rust allows both specifically in this case :upside_down_face: .

But I like *almost* everything else ruff is doing so I won't complain if this is automated.

Reply:
> Rust allows both specifically in this case 🙃 .

Oof, rust has all the features.

> But I like almost everything else ruff is doing so I won't complain if this is automated.

This one isn't actually (not everything is automated), but we could disable that rule